LondonBritishLibrary/orient/BLorient9798.xml

```diff
@@ -0,0 +1,281 @@
+                                    <q xml:lang="gez"><title>ሐሳበ፡ ተአምር፡ ጽንዕት፡ ወጽንጽንት፡</title> በመስከረም፡ ፲ወ፪ጽንዕት። ወላዕሉ፡ ጽንፅንት። በጥቅምት፡ ፳፫ጽንፅንት፡ ወላዕሉ፡ ጽንዕት፡</q>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus target="#49v"/>
+                                    <desc type="GuestText">A fragment of an Amharic work concerning the Trinity.</desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote">In the blank space between the columns a note partly illegible with the name of an 
+                                        British army officer as owner of the manuscript.</desc>
+                                    <q xml:lang="en"><persName role="owner" ref="PRS14353Edwards">William Wilson Edwards</persName>, 
+                                        Capt. of Madras Light Cavalry and Abyssinian Transport Train 
+                                        <date notBefore="1867" notAfter="1868">1867-1868</date></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to Emperor 
+                                        <persName role="owner" ref="PRS9142TaklaGi">Takla Giyorgis</persName>, 
+                                        who gave it to his sons.</desc>
+                                    <q xml:lang="gez">ዝሰዋስው፡ ዘ<persName role="owner" ref="PRS9142TaklaGi">ተክለ፡ ጊዮርጊስ፡ 
+                                        <roleName type="function">ንጉሥ፡</roleName></persName>  ዘወሀቦሙ፡ ለውሉዱ፡</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to a person with the name 
+                                        <persName role="owner" ref="PRS14501Yohannes">Yoḥannǝs</persName></desc>
+                                    <q xml:lang="gez">ዝሰዋስው፡ ዘ<persName role="owner" ref="PRS14501Yohannes">ዮሐንስ፡</persName></q>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#29ra #57r"/>
+                                    <desc type="Unclear">Scribbles and pen trials in red ink</desc>
+                                </item>
+                            </list>
+                        </additions>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1868">18th/19th century</origDate>
```

Comment:
Do we accept the information that this book is of Takla Giyorgis and was produced not later than his reign or before that? If so, the dating should be modified.

Reply:
Yes, I would accept and modified the information.